source/main/ipc/getHardwareWalletChannel.ts

```diff
@@ -420,9 +424,8 @@ export const handleHardwareWalletRequests = async (
 
   handleInitTrezorConnectChannel.onRequest(async () => {
     logger.info('[HW-DEBUG] INIT TREZOR');
-    resetTrezorListeners();
-    TrezorConnect.manifest(manifest);
     await initTrezorConnect();
+    resetTrezorListeners();
```

Comment:
This was wrong and stopped working with the update to 8.2.8, obviously listeners can only be attached once TrezorConnect is initialized. I'm unsure why it worked earlier.